pkg/webhook/server.go

```diff
@@ -87,25 +77,26 @@ func (s *Server) setDefaults() {
 	}
 }
 
-// Register validates and registers webhook(s) in the server
-func (s *Server) Register(webhooks ...Webhook) error {
-	// TODO(directxman12): is it really worth the ergonomics hit to make this a catchable error?
-	// In most cases you'll probably just bail immediately anyway.
-	for i, webhook := range webhooks {
-		// validate the webhook before registering it.
-		err := webhook.Validate()
-		if err != nil {
-			return err
-		}
-		_, found := s.webhooks[webhook.GetPath()]
-		if found {
-			return fmt.Errorf("can't register duplicate path: %v", webhook.GetPath())
-		}
-		s.webhooks[webhook.GetPath()] = webhooks[i]
-		s.webhookMux.Handle(webhook.GetPath(), webhook)
+// Register marks the given webhook as being served at the given path.
+// It panics if two hooks are registered on the same path.
+func (s *Server) Register(path string, hook http.Handler) {
+	_, found := s.webhooks[path]
+	if found {
+		panic(fmt.Errorf("can't register duplicate path: %v", path))
```

Comment:
In #300 I have re-order the code here a bit.
Moving `s.webhookMux.Handle(path, instrumentedHook(path, hook))` to the beginning. And it actually does the check for duplicate paths.

Reply:
handle does a check for duplicate paths itself, but the message might not be super-nice -- this presents us a way to have a slightly nicer message.